src/instructlab/model/evaluate.py

```diff
@@ -516,6 +518,43 @@ def launch_server(
     is_flag=True,
     help="Print serving engine logs.",
 )
+@click.option(
+    "--input-questions",
+    type=click.Path(exists=True, path_type=pathlib.Path),
+    default="",
+    help="Path to the questions and reference answers the model will be evaluating in the LLMaaJ evaluation",
+)
+@click.option(
+    "--output-file-formats",
+    type=click.STRING,
+    default="jsonl",
+    show_default=True,
+    help="Comma-separated list of file formats for results of the LLMaaJ evaluation. Valid options are csv, jsonl, and xlsx. If this option is not provided the results are written as a .jsonl file",
+)
+@click.option(
+    "--model-prompt",
+    type=click.STRING,
+    default=ragas_eval._DEFAULT_SYSTEM_PROMPT,
+    help="Prompt for the model when getting responses in the LLMaaJ evaluation",
+)
+@click.option(
+    "--temperature",
+    type=click.FLOAT,
+    default=0.0,
+    help="Temperature for the model when getting responses in the LLMaaJ evaluation",
+)
+@click.option(
+    "--model-name",
+    type=click.STRING,
```

Comment:
Should we call this `--model` instead of `--model-name`? I believe in other evaluations we just refer to it as `--model`.

Same for the judge model - should it just be `--judge-model` instead of `--judge-model-name`?

Reply:
^ It looks the above already exist as flags, we should just reuse those